src/index.ts

```diff
@@ -59,6 +81,31 @@ const main = async () => {
   console.error(`Initializing Dynatrace MCP Server v${VERSION}...`);
   const { oauthClientId, oauthClientSecret, dtEnvironment, dtPlatformToken, slackConnectionId } = dynatraceEnv;
 
+  // Test connection on startup
+  try {
+    console.error(`Testing connection to Dynatrace environment: ${dtEnvironment}...`);
+    await testDynatraceConnection(dtEnvironment, oauthClientId, oauthClientSecret, dtPlatformToken);
+    console.error(`Successfully connected to the Dynatrace environment at ${dtEnvironment}.`);
+  } catch (error: any) {
+    console.error(`Fatal: Could not connect to the Dynatrace environment.`);
+    if (isClientRequestError(error)) {
+      const e: ClientRequestError = error;
+      let additionalErrorInformation = '';
+      if (e.response.status == 403) {
+        additionalErrorInformation =
+          'Note: Your user or service-user is most likely lacking the necessary permissions/scopes for this API Call.';
+      }
+      console.error(
+        `Client Request Error: ${e.message} with HTTP status: ${
+          e.response.status
+        }. ${additionalErrorInformation} (body: ${JSON.stringify(e.body)})`,
```

Comment:
This slightly duplicates the error handler in https://github.com/dynatrace-oss/dynatrace-mcp/blob/93a3fecc9b7b29b590d5a3b31fea9878840d6ab0/src/index.ts#L88-L96 

Not going to block this, but I think we could refactor it.